core/node/auth/auth_impl.go

```diff
@@ -686,17 +686,19 @@ func (ca *chainAuth) checkMembership(
 	address common.Address,
 	spaceId shared.StreamId,
 	results chan<- bool,
+	errors chan<- error,
 	wg *sync.WaitGroup,
 ) {
 	log := dlog.FromCtx(ctx)
 	defer wg.Done()
 	isMember, err := ca.spaceContract.IsMember(ctx, spaceId, address)
 	if err != nil {
-		log.Warn("Error checking membership", "err", err, "address", address.Hex(), "spaceId", spaceId)
+		log.Error("Error checking membership", "err", err, "address", address.Hex(), "spaceId", spaceId)
```

Comment:
This could actually be downgraded to a warning as it will fire if another wallet evaluates to have membership, because we terminate remaining checks as soon as a single wallet is conclusively determined to be a member.